syntax = "proto3";

package flotilla;

service WorkerService {
  rpc Hint(HintRequest) returns (HintResponse) {}
  rpc Answer(AnswerRequest) returns (AnswerResponse) {}
}

message HintRequest {}

message HintResponse {
  bytes share = 1;
}

message AnswerRequest {
  bytes query = 1;
}

message AnswerResponse {
  bytes share = 1;
}
